### PR TITLE
Added IDE files to .gitignore

### LINKED ISSUE #12: Add IDE files to gitignore
Add the ide files to .gitignore file

1) `rm -rf .idea`
2)  delete these in `.gitignore`
```
/.idea/workspace.xml
/.idea/libraries
/.idea/misc.xml
```
3) Add this to `.gitignore`
```
.idea
```

Original issue [here](https://github.com/Njack-IITP/IITP-Connect/issues/44)

### PR DESCRIPTION
**Pull request by AK-007**
Fix #12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,8 @@
 *.iml
 .gradle
+rm -rf .idea
```
Comment (srv-twry) [inline .gitignore]: I don't think you need to add this in the ```.gitignore```
This was meant to be run on the command line first and then you should edit the ```.gitignore``` and do steps 2 and 3.
@mayank8318 views ?

Comment (mayank8318) [reply .gitignore]: Just keep plain `.idea/`

**Comment by ShikharJ** [comment]
@AK-007 @srv-twry If you guys can get the conflicts resolved, we can get this merged in and graded.

**Comment by srv-twry** [comment]
@ShikharJ This will be merged as soon as the travis build succeeds. Thanks :+1:

**Comment by sairamch04** [comment]
This doesn't work guys :) refer: https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore  https://github.com/NJACKWinterOfCode/IITP-Connect/pull/14#discussion_r155530650

**Comment by srv-twry** [comment]
@sairamch04 Thank you :) Will take care when i am pulling it to the main njack repo

**Pull request merged**